src/qurveros/optspacecurve.py

```diff
@@ -50,21 +51,26 @@ def __init__(self, *, curve, order, interval, params=None,
         self.params_history = None
         self.loss_grad = lambda params: None
 
+        #if params is not None:
```

Comment:
Please remove this comment and any other areas where you have commented out code- comments remaining in a PR should be for documentation purposes only.